### DISCUSSION TITLE
performance drop

### DISCUSSION THREAD

**Discussion by motorist828** [vladmandic/sdnext#1368] [Q&A]
I am using amd vega 56 8gb, ryzen 5 2600, 32gb ram, windows 10 22h2 build 19045
I have long been tormented by the problem of performance degradation after using the computer during the day (games, YouTube,discord)
The generation speed is reduced by 2-3 times, in comparison with what happens after turning on the computer
I thought that the fault was the use of video memory by various programs
I did a test, closed everything that could consume it, the video memory load was according to msi afterburner 570mb, this is almost the same value as after turning on the PC, but I still get very low speed
When generating, there are drawdowns in the load on the GPU

What could be causing this?

![Screenshot_175](https://github.com/vladmandic/automatic/assets/40582999/a9ebe7b1-0e5e-4b80-8390-b992e3632c2f)
![Screenshot_176](https://github.com/vladmandic/automatic/assets/40582999/aebf6ddf-1ef9-451d-8c30-9b4ce6dc5e1c)

**Comment by motorist828** [comment]
Looks like I also had another slowdown issue
Apparently, in one of the updates, the choice of the Live preview method was removed
not one of them was selected, after installing Approximate NN the speed returned

![Screenshot_177](https://github.com/vladmandic/automatic/assets/40582999/d164a72a-afc2-4118-a798-555084d28c18)

my normal speed after turning on the pc
![Screenshot_179](https://github.com/vladmandic/automatic/assets/40582999/c7a2e474-33f0-43c1-93b6-3142a3d01c4f)
![Screenshot_178](https://github.com/vladmandic/automatic/assets/40582999/91e6f15b-2d4e-4c1f-ad34-83de2a367389)

**Comment by aniketgore** [comment]
The control net is also not working since last update.

**Reply by vladmandic** [reply]
> The control net is also not working since last update.

not helpful as its in the wrong thread and has absolutely no info.

**Reply by aniketgore** [reply]
I get this error everytime i generate something or try to use controlnet

(ERROR) [.] Could not set ControlNet value: No module named 'extensions.sd-webui-controlnet'

**Reply by vladmandic** [reply]
still in the wrong THREAD? how is this related to "performance drop"????

**Reply by aniketgore** [reply]
Before this update I could generate a 768*768 image in 3 seconds, but now it takes 5 seconds. I've a linux ubuntu machine and rtx5000.

**Reply by aniketgore** [reply]
The control net is not working in auto as well. Guess it's the issue with the main repo.

**Reply by iDeNoh** [reply]
> The control net is not working in auto as well. Guess it's the issue with the main repo.

He's telling you that you need to start a new post, control net not working is not related to the topic of this post.
